docs/variants/clevo_nv41/hardware-matrix.md

```diff
@@ -28,6 +28,7 @@ coreboot port on the Clevo NV41 laptop.
 |                                | 5. Logitech, Inc. Keyboard K120                  |
 |                                | 6. 4-port USB Type-C Hub Pro                     |
```

Comment:
In my opinion, there's no sense in putting two different USB-C hubs in one HW matrix. Please remove from the table `4-port USB Type-C Hub Pro`.